styles/components/_overview-filters.scss

```diff
@@ -0,0 +1,21 @@
+.cmp-overview-filters {
+  width: 100%;
+  margin: space("sm") 0;
+  display: flex;
+  flex-wrap: wrap;
+
+  &__group {
+    display: flex;
+    flex-wrap: wrap;
+    gap: space("sm");
+    border: none;
+    padding: 0;
+    padding-top: space("xs");
+    margin-right: space("lg");
+    margin-bottom: space("sm");
+
+    &-label {
```

Comment:
Label styling isn't 1-to-1 with the Figma design, but since we're doing an overall styling pass in 193 I'm not going to let that block this.